src/common/common.csproj

```diff
@@ -32,6 +32,12 @@
   <PropertyGroup>
     <DefineConstants>$(DefineConstants);$(TelemetryType)</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net6.0|AnyCPU'">
+    <NoWarn>8002</NoWarn>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|net6.0|AnyCPU'">

```

Comment:
@Eneuman do we need this empty No warn with no codes in it ? May be this was missed.. :)

Reply:
That make sense :).